content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -138,8 +138,6 @@ different Kubernetes components.
 | `SupportPodPidsLimit` | `false` | Alpha | 1.10 | 1.13 |
 | `SupportPodPidsLimit` | `true` | Beta | 1.14 | |
 | `Sysctls` | `true` | Beta | 1.11 | |
-| `TaintBasedEvictions` | `false` | Alpha | 1.6 | 1.12 |
-| `TaintBasedEvictions` | `true` | Beta | 1.13 | |
```

Comment:
@ingvagabund, I think you want to **move** these lines into the section _Feature gates for graduated or deprecated features_.